### PR TITLE
Fixed modules/system py files for 2.4 to 3.5 exceptions

### PR DESCRIPTION
**Pull request by chris-l-weber**
##### ISSUE TYPE

- Feature Pull Request
##### COMPONENT NAME

System 
##### ANSIBLE VERSION

```
2.2
```
##### SUMMARY

Fixed system py files for 2.4 and 3.5 exception compatibility 
pycon2016 sprints

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @chris-l-weber for this PR. This module is maintained by the Ansible core team, so it can take a while for patches to be reviewed. Thanks for your patience.

Core team: please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with 'needs_revision' or merge as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

File: system/locale_gen.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 import os.path
 from subprocess import Popen, PIPE, call
 import re
+from ansible.module_utils.pycompat24 import get_exception
```
Comment (abadger) [inline system/locale_gen.py:22]: get_exception is being imported twice in this file.

**Comment by abadger** [comment]
@chris-i-weber This looks good, I found one place where get_exception is being imported twice but I can change that after I merge it.  Thanks!

**Comment by abadger** [comment]
Merged to devel and fix for get_exception double import here: d1174cc8b729aa00b9587b675aee788e0fd39035

**Comment by ttbigc** [comment]
After this merge I am now getting the following error when using the getent module:
Could not find imported module support code for getent. Looked for either get_exception or pycompat24

Any suggestions on how I should troubleshoot or what additional information needs to be provided to determine if this issue needs to be re-opened?

**Pull request merged**